### ISSUE TITLE
训练pp-liteseg模型 如何将RandomPaddingCrop取消，不进行随机裁剪

### ISSUE THREAD

**Issue by alanxinn** [PaddlePaddle/PaddleSeg#3778]
### 问题确认 Search before asking

- [X] 我已经搜索过问题，但是没有找到解答。I have searched the question and found no related answer.

### 请提出你的问题 Please ask your question

train_dataset:
  transforms:
    - type: ResizeStepScaling
      min_scale_factor: 0.125
      max_scale_factor: 1.5
      scale_step_size: 0.125
    - type: RandomPaddingCrop  #从原始图像和标注图像中随机裁剪1024x512大小
      crop_size: [1024, 512]
    - type: RandomHorizontalFlip
    - type: RandomDistort
      brightness_range: 0.5
      contrast_range: 0.5
      saturation_range: 0.5
    - type: Normalize

上述为默认config

若将crop_size: [1024, 512]设置为训练图像的真实尺寸，是否就不会进行裁剪了？ 或者直接将RandomPaddingCrop这个type注释掉即可

**Comment by changdazhou** [comment]
注释掉即可，但是并不绝对能跑通哈，需要实验一下，部分模型可能不支持任意分辨率图片输入

**Comment by alanxinn** [comment]
> 注释掉即可，但是并不绝对能跑通哈，需要实验一下，部分模型可能不支持任意分辨率图片输入

好的，我试一下，还有请教一下，将训练好的模型转为onnx 1x3x512x512 推理效果很好，但是转为1x3x256x256效果很差，如果我想训练的图resize到256x256，是否效果会变好呢？

**Comment by changdazhou** [comment]
会的，保证训练图尺寸和推理尺寸一致很重要的哈

**Comment by alanxinn** [comment]
> 会的，保证训练图尺寸和推理尺寸一致很重要的哈

好的，感谢您的耐心解答
